framework/signature-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
```

Comment:
I think I removed all bashisms. The script is working on both dash and bash now.